src/Domain/Creators/Python/BinaryCompactObject.hpp

```diff
@@ -0,0 +1,11 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <pybind11/pybind11.h>
+
+namespace domain::creators::py_bindings {
+// NOLINTNEXTLINE(google-runtime-references)
```

Comment:
I suggest that you insert a comment that for now you are hard-coding to the default values the time_dependent_options, outer_boundary_condition, and context parameters in the BinaryCompactObject constructor